### ISSUE TITLE
Browser history in data explorer doesn't seem to work

### ISSUE THREAD

**Issue by toolness** [18F/calc#1068]
Try doing the following in the data explorer (i.e., front page of CALC):

1. On a brand new visit in a brand new browser tab/window, the search field should be empty and all advanced options should be set to their default. There should be upwards of 50k results.

2. Type "engineer" into the search field and press <kbd>Enter</kbd>. There should now be about 7k results.

3. Press the browser's **back** button once. Note that while there is a brief loading animation on the front-end, nothing actually changes once the loading is finished: the search is still for "engineer" and the same number of results are shown. Even the URL hasn't changed. (One would expect the search to be for the empty string, and for 50k results to be shown.)

4. Press the **back** button again. Note that again, while a loading animation is shown, nothing changes once the loading is finished.

5. Press the **back** button a third time. Note that while the URL has changed--there is no longer a querystring in it--nothing else has. Our search is still for "engineer" and there are about 7k results.

This bug manifests itself on calc.gsa.gov, which I guess means it's been there since before CALC 2 started.

**Comment by toolness** [comment]
I think this line in `explorer.js`'s `popstate()` function is part of the problem:

```js
  // read the query string and set values accordingly
  const data = hourglass.extend(
    form.getData(),
    hourglass.qs.parse(location.search)
  );
```

Because we're not assigning defaults to the return value of `hourglass.qs.parse()`, it means that items left out of the querystring won't be in the object that we extend atop the form data, so the existing form data values will persist.  This explains why going from `/?exclude=1` to `/` via popstate doesn't do anything, though it doesn't explain why we have so many extra history entries.

**Comment by toolness** [comment]
This was just fixed by #1075! Woot!
